lib/polymorphic_integer_type/extensions.rb

```diff
@@ -95,7 +95,7 @@ def remove_type_and_establish_mapping(name, options, scope)
       def retrieve_polymorphic_type_mapping(polymorphic_type:, class_name:)
         return if polymorphic_type.nil?
 
-        belongs_to_class = class_name.safe_constantize
```

Comment:
Can we get a spec that might help exercise the changed code?

Reply:
That would be good. I'll look into it.